Makefile

```diff
@@ -127,7 +127,11 @@ check-tidy:
 .PHONY: build-image
 build-image:
 	# Add --pull so that we are using the latest base image.
-	docker build --pull --file ./cmd/$(TARGET)/Dockerfile --tag $(IMAGE_NAME) --build-arg GIT_HASH=$(GIT_HASH) .
+	docker build --pull \
+		--file ./cmd/$(TARGET)/Dockerfile \
+		--tag $(IMAGE_NAME) \
+		--platform linux/amd64,linux/arm64 \
+		--build-arg GIT_HASH=$(GIT_HASH) .
```

Comment:
The custombuild/Makefile also needs update. I am thinking if we can re-organize the Makefile structure, so that we do not have to maintain custombuild/Makefile anymore.

Reply:
Lets focus on multi platform build in this pr, and open another issue to merge Makefiles